app/src/main/res/layout/fragment_layout_not_found_data.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ImageView
+        android:id="@+id/image_not_founded"
+        android:layout_width="@dimen/dp_300"
+        android:layout_height="@dimen/dp_300"
```

Comment:
ko fix cứng size quá to thế này nhé , cả margin cũng thế. Dùng dimension ratio hoặc guide line mà neo